src/baldr/graphid.cc

```diff
@@ -1,9 +1,11 @@
 #include "baldr/graphid.h"
+#include "baldr/rapidjson_utils.h"
 
 namespace valhalla {
 namespace baldr {
 
 // The json representation of the Id
+// used in valhalla/mjolnir/util.h
```

Comment:
Not a must but `TileManifest::ToString()` could easily be refactored here as well, then this method can be dropped.